packages/sanity/src/core/form/inputs/arrays/ArrayOfPrimitivesInput/ArrayOfPrimitivesInput.tsx

```diff
@@ -19,6 +19,14 @@ import {ArrayOfPrimitivesFunctions} from './ArrayOfPrimitivesFunctions'
 export class ArrayOfPrimitivesInput extends React.PureComponent<ArrayOfPrimitivesInputProps> {
```

Comment:
Would be nice to add a type for the component state here:
```suggestion
export class ArrayOfPrimitivesInput extends React.PureComponent<
  ArrayOfPrimitivesInputProps,
  {disableTransition: boolean}
> {
```